docs/pages/setup/admin/daemon.mdx

```diff
@@ -143,15 +154,44 @@ To upgrade a host to a newer version of Teleport, you must:
 
 </Admonition>
 
-## Further reading
+
+## Understanding Teleport daemon command line options for unit file configuration
+
+The `teleport install systemd` command includes a number of optional flags that you can use to strictly define parameters of the generated unit file and set where that output will be written.
+The following table includes all command line options available with the `teleport install systemd` command, a brief description of what they do, and our general recommended defaults tested on **Ubuntu 20.04 LTS**:
+
+
+| Flag | Description | General Recommended Default |
+| --- | ----------- | ---- |
+| --fd-limit | The maximum number of open file descriptors, definted by `LimitNOFILE` in the unit file. |  **--fd-limit=8192** |
+| --env-file | The path to the `EnvironmentFile` defined by the system unit file, containing any potential variables to be used in the configuration. | **--envfile=/etc/default/teleport** |
+| --pid-file | The path to the pid file containing the process identification number. | **--pid-file=/run/teleport.pid** |
```

Comment:
Like the previous rows, I think it would make more sense to put flag examples in code style to make it explicit that these are possible parts of the `teleport install systemd` command.